.github/workflows/regression-tests.yaml

```
@@ -2,6 +2,7 @@ name: Regression Tests
 on:
   pull_request:
     types: [opened, synchronize, reopened, ready_for_review]
+  merge_group: { }
```

```suggestion
  merge_group:
    types: [checks_requested]
```

per https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#merge_group